lib/miq_expression/tag.rb

```diff
@@ -19,6 +19,7 @@ def self.parse(field)
 
   def initialize(model, associations, column, managed = true)
     super(model, associations, column)
+    @managed = managed
     @namespace = "/#{managed ? MANAGED_NAMESPACE : USER_NAMESPACE}/#{column}"
```

Comment:
@imtayadeway hmm not sure what you mean. Do you mean taking the namespace and translating it into the report_column?